docs/server_config/nginx.conf.dist

```diff
@@ -97,6 +97,12 @@ server {
         access_log off;
     }
 
+    location /admin/ { #Change this to your admin folder
```

Comment:
default is `admin-dev` maybe use github configuration?

Reply:
Was designed to work from a release (admin dir)